etc/nsjail/execute.cfg

```diff
@@ -10,7 +10,7 @@ log_level: FATAL
 rlimit_as_type: INF
 rlimit_cpu_type: SOFT
 rlimit_fsize: 1024 # 1GB
-rlimit_nofile: 100
+rlimit_nofile: 150
```

Comment:
this is based on a bit of experimentation, 125 was not enough, 150 was

Reply:
thanks... 100 was a bit made up before :)